core/templates/dev/head/pages/admin-page/admin-page.scripts.ts

```diff
@@ -16,7 +16,7 @@
  * @fileoverview Directives required in admin panel.
  */
 
-require('AppInit.ts');
+require('pages/admin-page/admin-page.module.ts');
 require('App.ts');
```

Comment:
It looks like this file doesn't match our other naming conventions. Is there a plan to rename or remove it in the future?

Reply:
Yes App.ts would be not needed in future since it is just a file of imports and AJS configs